### PR TITLE
Next revision

### PR DESCRIPTION
**Pull request by m-mohr**
Created this so that I can comment more easily on changes :-)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,22 +18,18 @@ This document explains the Classification Extension to the [SpatioTemporal Asset
 
 | Field Name              | Type                | Description |
 | ----------------------- | ------------------- | ----------- |
-| classification:classed  | `Classed`           | **REQUIRED**. Classes in the dataset |
+| classification:class_list  | `[string]`           | **REQUIRED**. Summary of class names in the dataset |
+
+| Field Name              | Type                | Description |
```
Comment (m-mohr) [inline README.md:23]: I'd suggest to merge the three tables :-)

File: README.md [hunk 2]
```diff
@@ -18,22 +18,18 @@ This document explains the Classification Extension to the [SpatioTemporal Asset
...
+| classification:classes  | `[Class]`         | **REQUIRED**. Categorized classes in the dataset |
 
 | Field Name               | Type               | Description |
 | ------------------------ | ------------------ | ----------- |
-| classification:bitmask   | `[Bitmask]`        | **REQUIRED**. Classes in the dataset |
-
-### Classed Object
-
-*Describes multiple classes*
+| classification:bitmask   | `[Bitmask_Part]`        | **REQUIRED**. Classes stored in bit ranges in the dataset |
```
Comment (m-mohr) [inline README.md:29]: Should they really all be required? I think I have use cases in mind where I don't need bitmasks.

File: README.md (hunk 1 above)
Comment (drwelby) [reply README.md:23]: yes, good idea!

File: README.md [hunk 3]
```diff
@@ -18,22 +18,18 @@ This document explains the Classification Extension to the [SpatioTemporal Asset
 
 | Field Name              | Type                | Description |
 | ----------------------- | ------------------- | ----------- |
-| classification:classed  | `Classed`           | **REQUIRED**. Classes in the dataset |
+| classification:class_list  | `[string]`           | **REQUIRED**. Summary of class names in the dataset |
```
Comment (m-mohr) [inline README.md:21]: I assume this is holding the same values as the name in classification:classes? Why do we need a summary then?

File: README.md (hunk 2 above)
Comment (drwelby) [reply README.md:29]: errr, no, that doesn't make sense

File: README.md (hunk 3 above)
Comment (drwelby) [reply README.md:21]: We were discussing that in #4, probably don't need it but I'm waiting for clarification from @pjhartzell

**Comment by m-mohr** [comment]
Some additional things:

What is the intention of:
> _`value: any` leaves it open for ranges but hopefully that can be discouraged!_

color-hint: 

> `RGB string or null`

I think it should be clarified what RGB string is. Is it e.g. `#ffffff` or `ffffff` or `rgb(255,255,255)` or ...?

role: What's that about? I don't really get it from looking at the referenced PR

**Comment by drwelby** [comment]
RGB string: agreed, if I can decide on one!

> value: any leaves it open for ranges but hopefully that can be discouraged!

That's a more of a note right now - we don't really intend this as a place to define breaks in a continuous value raster. We could be more strict at this point and say it has to be a singular type. I supposed we could be really mean and say this only should store `int`s.

**Comment by drwelby** [comment]
I feel like I've seen multiple roles, I'll check in on that.

`Hex RGB code in upper-case without leading #` works for me

Our immediate use cases seem to all be `ints` but there's been discussion of using this also for tabular data so they could also be strings. Would we disallow lists of ints if there are multiple values that map to the same class?

**Comment by pjhartzell** [comment]
Landsat bitmask raster Assets have multiple roles, e.g. , the QA_PIXEL raster contains info on clouds, water, and snow/ice. The USGS lists 4 roles: ["cloud", "cloud-shadow", "snow-ice", "water-mask"]

File: README.md (hunk 3 above)
Comment (pjhartzell) [reply README.md:21]: @drwelby I'm good with removing `classification:class_list`

**Comment by m-mohr** [comment]
I think we should also add some language around where you can use it and what is recommended. raster:bands? eo:bands? item properties? assets? ...

**Comment by pjhartzell** [comment]
I'm interested to hear others' thoughts on this. As someone new to STAC, I look at eo:bands at face value (no context to its history) as an extension to describe spectral bands. Since classification is a derivative product, I lean toward recommending it for raster:bands.

That said, eo:bands seems to be used quite a bit for bands containing non-spectral data, so it should be available for use in eo:bands. 

Should be available for use in assets (for single band image assets that do not use the eo or raster extensions) and item properties (instances where all assets use the same classification scheme) as well, no?

**Comment by m-mohr** [comment]
> I'm interested to hear others' thoughts on this. As someone new to STAC, I look at eo:bands at face value (no context from its history) as an extension to describe spectral bands. Since classification is a derivative product, I lean toward recommending it for raster:bands.

I guess we should allow it for item properties and raster:bands. Allowing it in item properties implicitly also allows them in assets and Collection summaries.

> That said, eo:bands seems to be used quite a bit for bands containing non-spectral data, so it should be available for use in eo:bands.

Although I'd say we should encourage the discouraged. non-spectral data should not be in eo:bands although some people di it...

**Comment by m-mohr** [comment]
I've added a proposal for where it can be used. Merging now so that we have the current status on main again and can do more fine-grained PRs against it.

**Pull request merged**